src/lib/zf/cplane.c

```diff
@@ -182,7 +182,10 @@ __zf_cplane_get_path(struct zf_stack* st, struct zf_path* path,
   struct cp_fwd_data data;
   cicp_verinfo_t verinfo;
   int rc;
-  int iter;
+  int time_limit;
+  int time_used;
```

Comment:
Admittedly half an hour would be a long timeout but the limit of these integral types is a little close for comfort to times that could be seen in the real world, e.g. if you were using a simulation platform but measuring real time. I suggest a 64-bit quantity would be more appropriate for microseconds? Maybe use `time_t`?